src/graph/test/DataTest.cpp

```diff
@@ -0,0 +1,285 @@
+/* Copyright (c) 2019 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#include "base/Base.h"
+#include "graph/test/TestEnv.h"
+#include "graph/test/TestBase.h"
+#include "meta/test/TestUtils.h"
+
+DECLARE_int32(load_data_interval_secs);
+
+namespace nebula {
+namespace graph {
+
+class DataTest : public TestBase {
```

Comment:
Maybe InsertionTest?

Reply:
It will test all data operation, not only insertion